test/start-kubernetes.sh

```diff
@@ -0,0 +1,240 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+
+TEST_DIRECTORY=${TEST_DIRECTORY:-$(dirname $(readlink -f $0))}
+TEST_CONFIG=${TEST_CONFIG:-${TEST_DIRECTORY}/test-config.sh}
+source ${TEST_CONFIG}
+DEPLOYMENT_SUFFIX=${DEPLOYMENT_SUFFIX}
+DEPLOYMENT_ID=${DEPLOYMENT_ID:-k8s-test${DEPLOYMENT_SUFFIX}-pmem}
```

Comment:
Instead of `k8s-test` let's use something more descriptive.

I suggest `pmem-csi-<OS[-suffix]>`, which then matches with a `_work/<OS[-suffix]-govm` subdirectory.

Reply:
I updated the name